MetaMask/metamask-extension
# fix: minor insights v2 tweaks

## **Description**



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23248?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2124

## **Manual testing steps**

1. Go to https://metamask.github.io/test-dapp
2. Connect your Flask Wallet
3. Call "Malicious ERC20 transfer"
4. Confirm the transaction.

## **Screenshots/Recordings**

### **Before**
![beforeDark](https://github.com/MetaMask/metamask-extension/assets/44679989/fe5cafb2-264d-4394-839a-423fd48ffcc6)
![beforeLight](https://github.com/MetaMask/metamask-extension/assets/44679989/937a56cf-be48-4767-8b00-bd415ed4f65c)

### **After**
<img width="360" alt="afterLight" src="https://github.com/MetaMask/metamask-extension/assets/44679989/b60b0b5a-db6a-423e-8718-5d77ef739f59">
<img width="358" alt="afterDark" src="https://github.com/MetaMask/metamask-extension/assets/44679989/0a8ba6e7-875b-404a-a00d-a494cba9872b">

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
